app/assets/javascripts/i18n.js

```diff
@@ -55,6 +55,11 @@
     return type === 'function' || type === 'object' && !!obj;
   };
 
+  // Check if value is different than undefined and null;
+  var isSet = function(value) {
+    return value !== undefined && value !== null;
```

Comment:
I think the common check should be `typeof value !== "undefined" && value !== null`
This avoids accident when `undefined` assigned to something else